axelrod/tournament.py

```diff
@@ -310,8 +310,8 @@ class ProbEndTournament(Tournament):
 
     def __init__(self, players, match_generator=ProbEndRoundRobinMatches,
                  name='axelrod', game=None, prob_end=.5, repetitions=10,
-                 processes=None, prebuilt_cache=False, noise=0,
-                 with_morality=True):
+                 processes=None, deterministic_cache=None, prebuilt_cache=False,
```

Comment:
At the moment: yes, the tournament manager and factory classes I believe
make use of `prebuilt_cache`. I'm being a bit lazy with it and just not
touching those two classes for now... (I see the plan being for
`prebuilt_cache` to go eventually, it's superceded by the ability to just
give a cache or not).

On Wed, 20 Apr 2016, 04:15 Marc Harper, <notifications@github.com> wrote:

> In axelrod/tournament.py
> <https://github.com/Axelrod-Python/Axelrod/pull/556#discussion_r60345423>:
>
> > @@ -310,8 +310,8 @@ class ProbEndTournament(Tournament):
> >
> >      def __init__(self, players, match_generator=ProbEndRoundRobinMatches,
> >                   name='axelrod', game=None, prob_end=.5, repetitions=10,
> > -                 processes=None, prebuilt_cache=False, noise=0,
> > -                 with_morality=True):
> > +                 processes=None, deterministic_cache=None, prebuilt_cache=False,
>
> Is it necessary to have both deterministic_cache and also prebuilt_cache?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Axelrod-Python/Axelrod/pull/556/files/e1a18bb1c0ed74e7ed8b909d68018a1792665549#r60345423>
>